keanu-project/src/main/java/io/improbable/keanu/vertices/LogProbGraphGenerator.java

```diff
@@ -0,0 +1,7 @@
+package io.improbable.keanu.vertices;
+
+public interface LogProbGraphGenerator {
```

Comment:
In java functional lingo this is a Suppiler of LogProbGraph. So maybe call this LogProbGraphSupplier?

Reply:
Ah, that was the name I was looking for